net/lp_rpc.proto

```diff
@@ -108,16 +108,25 @@ message Capabilities {
     // Capacity corresponding to each capability
     map<uint32, uint32> capacities = 3;
 
-    // Non-binary capability constraints, such as supported ranges.
+    string version = 4;
+
+    Constraints constraints = 5;
+
+    // Non-binary general constraints.
     message Constraints {
```

Comment:
@victorges To ensure a non-breaking merge, I created a new `CapabilityConstraints` message within the `Capabilities` message. I reused the `Constraints` message you added for general constraints that apply to all capabilities, while `CapabilityConstraints` are specific to individual capabilities. If you have any suggestions for improving the naming, please let me know.